src/main/java/cceclipseplugin/ui/dialogs/AddNewUserDialog.java

```diff
@@ -71,11 +73,14 @@ protected Control createDialogArea(Composite parent) {
 		
 		permissionMap = PluginManager.getInstance().getDataManager().getSessionStorage().getPermissionConstants();
 		BiMap<Byte, String> inversePermissionMap = permissionMap.inverse();
-		@SuppressWarnings("unchecked")
 		List<Byte> permissionCodes = new ArrayList<>(permissionMap.values());
+		HashMap<String, Permission> userPermissions = selectedProject.getPermissions();
+		int userLevel = userPermissions.get(PluginManager.getInstance().getDataManager().getSessionStorage().getUsername()).getPermissionLevel();
```

Comment:
You should get the reference to SessionStorage on line 73 and re-use that reference for lines 74 and 78 so that there aren't so many successive function calls.